jdk/src/share/classes/java/lang/StringCoding.java

```diff
@@ -92,7 +98,11 @@ private static char[] safeTrim(char[] ca, int len,
     private static int scale(int len, float expansionFactor) {
         // We need to perform double, not float, arithmetic; otherwise
         // we lose low order bits when len is larger than 2**24.
-        return (int)(len * (double)expansionFactor);
+        double result = len * (double) expansionFactor;
```

Comment:
Please don't change the formatting of the cast:
```java
        double result = len * (double)expansionFactor;
```

Reply:
Thanks Keith, I removed the added space.